syntax = "proto3";

package chatservice;

// protoc friend.proto --cpp_out=./

option cc_generic_services = true;

message ResultCode {
    int32 errcode = 1;
    bytes errmsg = 2;
}

message GetFriendsListRequest {
    uint32 userid = 1;
}

message GetFriendsListResponse {
    ResultCode result = 1;
    repeated bytes friends = 2;
}

// 好友模块
service FriendServiceRpc {
    // 注意，这里定义的方法名和本地服务的方法名不一定要一致，但是最好一致
    rpc GetFriendsList(GetFriendsListRequest) returns(GetFriendsListResponse);
}